### PR TITLE
CY-1336 - Add common scripts

### PR DESCRIPTION
**Pull request by qooban**

### CODE REVIEW

File: scripts/upload-package.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    MANAGER_USER=${MANAGER_USER:-centos}
+    PACKAGE_PATH=$1
+    SSH_COMMAND=$2
+    COMMON_OPTIONS="-o StrictHostKeyChecking=no"
+    ERROR_CODE=0
+
+    pwd
+
+    # Manager IP check
+    if [ -z $MANAGER_IP ]; then
```
Comment (tyacbovi) [inline scripts/upload-package.sh]: I will just verify it in one if and print out what the user entered

File: scripts/upload-package.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash -e
+
+#===================================================================================
```
Comment (tyacbovi) [inline scripts/upload-package.sh]: This is for dev purposes or CI?

Comment (qooban) [reply scripts/upload-package.sh]: The script itself? It's duplicated, very similar in stage and composer. Used in both - CI (system tests) and development.

Or the comment? I tried to find some style guide for bash scripts how to document them. It's not standardized, but that way it looks fine imo.

File: scripts/upload-package.sh (hunk 1 above)
Comment (qooban) [reply scripts/upload-package.sh]: Don't understand. Verify MANAGER_IP, SSH_KEY_PATH and PACKAGE_PATH in one if?

File: scripts/upload-package.sh (hunk 2 above)
Comment (tyacbovi) [reply scripts/upload-package.sh]: The script :)

File: scripts/upload-package.sh (hunk 1 above)
Comment (tyacbovi) [reply scripts/upload-package.sh]: yes, because you don't want to run the script multiple of times to find out that you had forgotten to fill in one of the three

File: scripts/upload-package.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        ((ERROR_CODE++))
+    fi
+
+    if [ $ERROR_CODE -gt 0 ]; then
+        exit $ERROR_CODE
+    fi
+
+    scp -i ${SSH_KEY_PATH} ${COMMON_OPTIONS} ${PACKAGE_PATH} ${MANAGER_USER}@${MANAGER_IP}:~
+
+    ssh -i ${SSH_KEY_PATH} ${COMMON_OPTIONS} ${MANAGER_USER}@${MANAGER_IP} ${SSH_COMMAND}
```
Comment (tyacbovi) [inline scripts/upload-package.sh]: Why is this needed?

File: scripts/upload-package.sh (hunk 1 above)
Comment (qooban) [reply scripts/upload-package.sh]: You don't have to run it multiple times. Exit is in the last if checking error code.

File: scripts/upload-package.sh (hunk 3 above)
Comment (qooban) [reply scripts/upload-package.sh]: To run command remotely (eg. unpack package, stop stage/composer app, change files, change permissions, etc.).
You can check uploadPackage.sh in stage/composer for the origins of the common one.

File: scripts/upload-package.sh (hunk 1 above)
Comment (tyacbovi) [reply scripts/upload-package.sh]: yes, but also a single msg for all three will provide the same info for using this script
